Sources/Plasma/FeatureLib/pfJournalBook/pfJournalBook.cpp

```diff
@@ -3100,6 +3437,25 @@ void    pfJournalBook::IFindFontProps( uint32_t chunkIdx, ST::string &face, uint
         spacing = 0;
 }
 
+//// IFindTextLink ///////////////////////////////////////////////////////////
+// Starting at the given chunk, works backward to determine the current text link ID
+pfEsHTMLChunk* pfJournalBook::IFindTextLink(uint32_t chunkIdx) const
+{
+    if (fHTMLSource.empty())
+        return nullptr;
+
+    for (size_t i = chunkIdx; i > 0; --i) {
```

Comment:
`i > 0` as the loop condition means that the loop body will never run for index 0 - is that intentional?

If the condition is supposed to be `i >= 0`, you'd also have to change the type of `i` to `hsSsize_t` (like the other loops above) to prevent an infinite loop.